pom.xml

```diff
@@ -134,6 +134,31 @@
             </resource>
         </resources>
 
+        <pluginManagement>
```

Comment:
Unfortunately if you wanna make the project compatible with eclipse, m2e plugin requires that this custom maven call needs to be inside pluginManagement.